go/worker/common/committee/group.go

```diff
@@ -255,6 +255,16 @@ type Group struct {
 	logger *logging.Logger
 }
 
+// RoundContext returns the active epoch's round context.
+func (g *Group) RoundContext() (*context.Context, error) {
```

Comment:
`context.Context` is an interface and so it can be nil, no need to return a pointer to it.